config.py

```diff
@@ -0,0 +1,4 @@
+class Config:
+    @staticmethod
+    def get_connection_string():
+        return 'DefaultEndpointsProtocol=[http|https];AccountName=mylibrary030;AccountKey=k+HGhydwZIsniLJtVJA+j+4rormfbo5AuPaxcRB8gOqUMBSRIWyPE1MXdoEhEY5Qe7QReSqps3rjKM4UpcpN+w=='
```

Comment:
The connection string is sensitive, should not appear in your PR.
Note: never submit your username, password, access key...